op-guide/configuration.md

```diff
@@ -45,8 +45,145 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 ### --perfschema
 + enable(1) or disable(0) the performance schema
 + default: "0"
-+ The value can be (1) or (0). (1) is to enable and (0) is to disable. The Performance Schema provides a way to inspect internal execution of the server at runtime. See [performance schema](http://dev.mysql.com/doc/refman/5.7/en/performance-schema.html) for more information. If you enable the performance schema, the performance will be affected.
++ the value can be (1) or (0). (1) is to enable and (0) is to disable. The Performance Schema provides a way to inspect internal execution of the server at runtime. See [performance schema](http://dev.mysql.com/doc/refman/5.7/en/performance-schema.html) for more information. If you enable the performance schema, the performance will be affected.
 
 ### $TIDB_PPROF environment variable
 + An environment variable that is used to enable or disable the runtime profiling data via the HTTP server. The Address is at client URL + "/debug/pprof/".
 + If set $TIDB_PPROF to 0, TiDB will disable pprof. Otherwise TiDB will enable pprof.
+
+## Placement Driver (PD)
+
+### -L
+
++ the log level
++ default: "info"
++ You can choose from debug, info, warn, error, or fatal.
+
+### -c 
+
++ the config file
++ default: ""
```

Comment:
I think we should add comment to let user to make sure that the cmd will replace the config set value for the same flag.